packages/model-viewer/src/test/model-viewer-base-spec.ts

```diff
@@ -99,6 +99,30 @@ suite('ModelViewerElementBase', () => {
       });
     });
 
+    suite('with modelViewer global config', () => {
+      ModelViewerElementBase.powerPreference = 'low-power'
+      let element: ModelViewerElementBase;
+
+      setup(() => {
+        element = new ModelViewerElement();
+        document.body.insertBefore(element, document.body.firstChild);
+      });
+
+      teardown(() => {
+        if (element.parentNode != null) {
+          element.parentNode.removeChild(element);
+        }
```

Comment:
This is fine, but especially in tests I'll often use the shorthand `element.parentNode!.removeChild(element);` since that's just telling TS I'm sure it exists. If it doesn't, the test will fail anyway.

Reply:
thanks for sharing.